src/Microsoft.Diagnostics.Monitoring.WebApi/Exceptions/ExceptionsOptions.cs

```diff
@@ -22,5 +23,7 @@ internal sealed class ExceptionsOptions :
         [Range(1, int.MaxValue)]
         [DefaultValue(ExceptionsOptionsDefaults.TopLevelLimit)]
         public int? TopLevelLimit { get; set; }
+
+        public ExceptionsConfiguration Filters { get; set; }
```

Comment:
`CollectionFilters` to make it clear that these are filters are applied at collection time.